src/Model/ModelTrait.php

```diff
@@ -65,7 +65,11 @@ protected function loadData(array $data, bool $update = false, array $replaceCol
                     // in this case there should be an "addProperty" method on the class
                     $adder = "add$prop";
 
-                    if ($param->getType() instanceof \ReflectionNamedType && method_exists($this, $adder)) {
+                    $paramType = $param->getType();
+                    if ($paramType instanceof \ReflectionNamedType &&
```

Comment:
If we're splitting to multi-line, this first condition should also be on it's own line

Reply:
Fixed